tools/code_format/tests/test_python_check.py

```diff
@@ -186,37 +188,41 @@ def test_python_check_recurse():
     assert "recurse" not in checker.__dict__
 
 
-def test_python_check_yapf(patches):
+@pytest.mark.asyncio
+async def test_python_check_yapf(patches):
     checker = python_check.PythonChecker("path1", "path2", "path3")
     patched = patches(
-        "PythonChecker.yapf_run",
+        "aio",
+        ("PythonChecker.yapf_format", dict(new_callable=MagicMock)),
+        "PythonChecker.yapf_result",
         ("PythonChecker.yapf_files", dict(new_callable=PropertyMock)),
         prefix="tools.code_format.python_check")
+    _files = ["file1", "file2", "file3"]
```

Comment:
OCD: `_files` local var.

Reply:
i have gone through and stripped all of the `_` prefixes